dask/base.py

```diff
@@ -816,3 +799,64 @@ def _colorize(t):
     h = hex(int(i))[2:].upper()
     h = '0' * (6 - len(h)) + h
     return "#" + h
+
+
+named_schedulers = {
```

Comment:
Are these all names that have been used historically? Of course, we'd prefer to keep the list concise, if possible, as this is what shows up in the error message should the user mistype the scheduler argument.

Reply:
These names are up for discussion.  What should they be?

Reply:
I think I'd keep `sync` as an alias for `synchronous`, and replace `threaded` with `threading`. Then we match joblib conventions (which others might be familiar with), while also adding `processes`, `threads`, and `synchronous`/`sync` (which I slightly prefer).

Reply:
All these aliases seems a little error prone to me(but probably harmless).

I would consider introducing top level constants (or maybe an enum), e.g., `dask.SYNCHRONOUS`, `dask.THREADING`, `dask.MULTIPROCESSING` and `dask.DISTRIBUTED`. With auto-complete, these are easier to type correctly than strings.